poll/poll.py

```diff
@@ -811,31 +835,33 @@ def vote(self, data, suffix=''):
         choices = self.get_choices()
         if choices and not self.private_results:
             result['success'] = False
-            result['errors'].append("You have already voted in this poll.")
+            result['errors'].append(self.ugettext("You have already voted in this poll."))
 
         if not choices:
             # Reset submissions count if choices are bogus.
             self.submissions_count = 0
 
         if not self.can_vote():
             result['success'] = False
-            result['errors'].append('You have already voted as many times as you are allowed.')
+            result['errors'].append(self.ugettext('You have already voted as many times as you are allowed.'))
 
         # Make sure the user has included all questions, and hasn't included
         # anything extra, which might indicate the questions have changed.
         if not sorted(data.keys()) == sorted(questions.keys()):
             result['success'] = False
             result['errors'].append(
-                "Not all questions were included, or unknown questions were "
-                "included. Try refreshing and trying again."
+                self.ugettext(
+                    "Not all questions were included, or unknown questions were included. "
+                    "Try refreshing and trying again."
+                )
             )
 
         # Make sure the answer values are sane.
         for key, value in data.items():
             if value not in answers.keys():
                 result['success'] = False
                 result['errors'].append(
-                    "Found unknown answer '%s' for question key '%s'" % (key, value))
+                    self.ugettext("Found unknown answer '%s' for question key '%s'") % (key, value))
```

Comment:
@asadiqbal08 This string should use named placeholders (`answer_key`, `question_key`) and is missing a translator comment, something like:

```python
# Translators: {answer_key} uniquely identifies a specific answer belonging to a poll or survey.
# {question_key} uniquely identifies a specific question belonging to a poll or survey.
```